openrtb_ext/utils.go

```diff
@@ -0,0 +1,91 @@
+package openrtb_ext
```

Comment:
Could we move `openrtb_ext/utils.go` into `exchange`? This package is intended just to define the API contracts.

Reply:
Yes, that would be fine. Do you want this branch updated, or place it in a new PR?

Reply:
This branch, for changes which address PR comments. New PRs for extra features.